linkerd/tracer/zipkin/src/main/scala/io/l5d/zipkin.scala

```diff
@@ -0,0 +1,34 @@
+package io.l5d
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.tracing.{Record, TraceId, Tracer}
+import com.twitter.finagle.zipkin.thrift.{Sampler, ZipkinTracer}
+import io.buoyant.linkerd.{TracerConfig, TracerInitializer}
+
+class ZipkinTracerInitializer extends TracerInitializer {
+  val configClass = classOf[zipkin]
+}
+
+object ZipkinTracerInitializer extends ZipkinTracerInitializer
+
+case class zipkin(
+  host: Option[String],
+  port: Option[Int],
+  sampleRate: Option[Double]
+) extends TracerConfig {
+
+  @JsonIgnore
+  override def newTracer(): Tracer = new Tracer {
+
+    private[this] def getHost: String = host.getOrElse("localhost")
+    private[this] def getPort: Int = port.getOrElse(9410)
+    private[this] def getSampleRate: Float =
```

Comment:
tioli: i don't think it adds much value to have `def getFoo` given that these just get called once, immediately.  Could even inline them into ZipkinTrace.mk...

nbd one way or the other